lucene/backward-codecs/src/test/org/apache/lucene/index/TestBackwardsCompatibility.java

```diff
@@ -168,7 +169,7 @@ public void testCreateMoreTermsIndex() throws Exception {
     Thread.sleep(100000);
   }
 
-  // ant test -Dtestcase=TestBackwardsCompatibility -Dtestmethod=testCreateSortedIndex -Dtests.codec=default -Dtests.useSecurityManager=false -Dtests.bwcdir=/tmp/sorted
+  // gradlew -p lucene/backward-codecs test --tests *testCreateSortedIndex* -Ptests.bwcdir=/tmp/sorted -Ptests.codec=default -Ptests.useSecurityManager=false
```

Comment:
should be --tests TestBackwardsCompatibility.testCreateSortedIndex.
A prefix wildcard may be needed (*TestBackwards...)?